Makefile

```diff
@@ -7,36 +7,45 @@ clean:
 	rm -rf build dist
 	find . -name '*.pyc' -exec rm \{\} \;
 
-install:
+deps:
+	pip install --upgrade pip build twine setuptools
+
+install: deps
 	@# Install OpenFisca-Aotearoa for development.
 	@# `make install` installs the editable version of OpenFisca-Aotearoa.
 	@# This allows contributors to test as they code.
-	pip install --upgrade pip
-	poetry install --sync
+	pip install --editable .[dev] --upgrade
+
+build: clean deps
+	@# Install OpenFisca-Extension-Template for deployment and publishing.
+	@# `make build` allows us to be be sure tests are run against the packaged version
+	@# of OpenFisca-Extension-Template, the same we put in the hands of users and reusers.
+	python -m build
+	find dist -name "*.whl" -exec pip install --force-reinstall {}[dev] \;
 
 check-syntax-errors:
 	python -m compileall -q .
 
-format:
+format-style:
 	@# Do not analyse .gitignored files.
 	@# `make` needs `$$` to output `$`. Ref: http://stackoverflow.com/questions/2382764.
-	poetry run isort `git ls-files | grep "\.py$$"`
-	poetry run autopep8 `git ls-files | grep "\.py$$"`
-	poetry run pyupgrade --py39-plus `git ls-files | grep "\.py$$"`
+	isort `git ls-files | grep "\.py$$"`
+	autopep8 `git ls-files | grep "\.py$$"`
+	pyupgrade --py39-plus `git ls-files | grep "\.py$$"`
 
-lint: clean check-syntax-errors format
+check-style:
```

Comment:
**suggestion**: stick with the terms `format` and `lint`, those are standard. "format-style" and "check-style" aren't immediately obvious, but most developers already use `format` and `lint`

Reply:
Thanks and added as suggested.